cfg.sample.toml

```diff
@@ -10,6 +10,23 @@ access_token = ""
 app_client_id = ""
 app_client_secret = ""
 
+# A base shared secret; this can be used to generate the per-repo github secret,
+# which is what's used to verify authenticity of github -> Homu webhooks. The
+# per-repo secret can be computed as follows (you'll need to enter this in
+# the repo webhook configuration):
+#
+# $ echo -n "$owner.$name" | openssl sha1 -hmac "$secret"
```

Comment:
I won't claim deep crypto experience either, but I'm not a novice either 😄 

What we're doing here is symmetric with how [Github uses HMAC](https://developer.github.com/webhooks/securing/).  The internet has text about HMAC - [this stackoverflow](http://security.stackexchange.com/questions/20129/how-and-when-do-i-use-hmac) seems decent.

I don't think we need a *key* here - we're not actually using this as a key for a cipher (whether symmetric or not).  We simply need a shared secret between 3 parties - Homu, Github, and the repo owner.  We don't want different repo owners to have the same secret.

Basically, why do you think we need a KDF here, but Github's use of HMAC to sign their requests is OK?  Or do you disagree with that as well?